cmd/net-state/main.go

```diff
@@ -1,27 +1,37 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
 package main
 
 import (
-	"fmt"
+	"log"
 	"net/http"
 
 	"github.com/julienschmidt/httprouter"
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
 
-	"github.com/storj/routes"
+	"github.com/storj/netstate/routes"
 	"github.com/storj/storage/boltdb"
+)
 
-	"go.uber.org/zap"
+var (
+	// Error is the default main errs class
+	Error = errs.Class("main err")
+
+	errLoggerFail = Error.New("zap logger failed to start")
```

Comment:
if you do Error.New when the error happens instead of here, the stacktrace will be generated when the error happens. as it stands, the stacktrace this error captured isn't useful. essentially, i'd make a LoggerFailure error class, and then do LoggerFailre.New when it happens